src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -23,14 +23,17 @@
 def handle_exception(ex):
     # For error code, follow guidelines at https://docs.python.org/2/library/sys.html#sys.exit,
     from msrestazure.azure_exceptions import CloudError
-    from msrest.exceptions import HttpOperationError
+    from msrest.exceptions import HttpOperationError, ValidationError
     from azure.cli.core.azlogging import CommandLoggerContext
 
     with CommandLoggerContext(logger):
 
         if isinstance(ex, (CLIError, CloudError)):
             logger.error(ex.args[0])
             return ex.args[1] if len(ex.args) >= 2 else 1
+        if isinstance(ex, ValidationError):
+            logger.error('validation error: %s', ex)
+            return 1
```

Comment:
```suggestion
            return 1
            
       if isinstance(ex, ClientRequestError):
            logger.error("The request failed:\n")
            logger.error(ex)
            return 1
```

Reply:
Some suggestions to handle `HttpConnection/ClientRequestError`s

see the following issues:
#9089, #9196,